api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -556,4 +556,12 @@ public void acceptTermsOfService() {
     TermsOfServiceApi termsOfServiceApi = termsOfServiceApiProvider.get();
     retryHandler.run((context) -> termsOfServiceApi.acceptTermsOfService(TERMS_OF_SERVICE_BODY));
   }
+
+  @Override
+  public boolean getUserTermsOfServiceStatus() throws ApiException {
```

Comment:
i can rename the method to getTerraTermsOfServiceStatus as well. Thoughts?

Reply:
I like this name.  We know it's Terra because it's FireCloudService.